addon/components/virtual-each/component.js

```diff
@@ -143,6 +136,10 @@ const VirtualEachComponent = Component.extend(EventListenerMixin, DefaultAttrsMi
 
   calculateVisibleItems(positionIndex) {
     run(() => {
```

Comment:
This method is called from `run.throttle` or `run.scheduleOnce` so I don't think this `run` wrapper is needed, but I'm not 100% sure that it can be removed. @jasonmit any thoughts?